packages/jsii-pacmak/lib/targets/java.ts

```diff
@@ -835,75 +844,110 @@ class JavaGenerator extends Generator {
         }
 
         collectProps(ifc);
+        const props = Array.from(propsByName.values());
```

Comment:
If you go for the suggestion against `Map`, you'd want to use `Object.values(propsByName);` here.